app/uk/gov/hmrc/tariffclassificationfrontend/views/refer_case.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.forms.ReleaseCaseForm
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case, Queue}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.release_case_form
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+
+    @case_heading(c)
+
+    <div class="grid-row">
+        <div class="column-full">
+            <h3 class="heading-large mt-0">Refer this case</h3>
+
+            <p> Before you refer this case, <a href="#">add a note</a> explaining why you’re referring it. </p>
```

Comment:
Are we adding this line in too early? The link should link to somewhere surely

Reply:
it was a requirement on the criteria. I could potentially add it but it was explicitly required